README.md

```diff
@@ -2,55 +2,158 @@
+var isTheGrassGreen = false
+isTheGrassGreen = true
+```
+
+In both of these examples, the Swift compiler has inferred the type from our use of the literal values, but we can also explicitly write out the type if we want to:
+
+```swift
+let canDolphinsWalk: Bool = false
+```
+
+## Comparing values
+
+A common use of the `Bool` type is as a result of a comparison of other values. Comparing two values in Swift is done with an *operator*, which is a special symbol that you use to check, combine or change one or more values.
+
+For example, we can compare the two integer variables from our earlier example, to see if they are equal, using the equal-to operator:
+
+```swift
+let doWeHaveAsManyApplesAsOranges = numberOfApples == numberOfOranges // false, because 7 is not equal to 12
+```
+
+The result in this case is of course `false`, because 7 and 12 are not equal numbers. We can also determine if two values are **not** equal, like this:
+
+```swift
+let compareApplesToOranges = numberOfApples != numberOfOranges // true, because 7 is indeed not equal to 12
+```
+
+In this case, the result is `true`. We may also be interested in learning if we have more apples than oranges, for this we have operators for greater and less than:
+
+```swift
+let moreApplesThanOranges = numberOfApples > numberOfOranges // false, because 7 is not greater than 12
+let lessApplesThanOranges = numberOfApples < numberOfOranges // true, because 7 is less than 12
+```
+
+Finally, there are also comparison operators for greater and less than or equal. Those will either be true if the two operands are equal or if the left hand side is greater, or respectively less, than the right hand side.
+
+```swift
+let lessOrTheSameNumberOfApplesAndOranges = numberOfApples <= numberOfOranges // true, because 7 is less than 12
+
+numberOfApples = 12
+let moreOrTheSameNumberOfApplesAndOranges = numberOfApples >= numberOfOranges // true, because 12 is not greater than 12, but 12 is equal to 12
+```
+
+These comparisons might not seem too useful on their own, but you will learn later how you can use them in *conditional expressions* to execute different statements in your programs, depending on the value of these kind of conditions.
+
+## Logical operations
+
+In the last example, we already saw a simple logical statement for our program, we were able to determine if two values are greater than or equal to each other. Boolean values can also be combined using *logical operators* to formulate more complex logical statements about variables. 
+
+The logical `NOT` operator inverts the value of a Boolean, so that `true` becomes `false` and vice versa. This operator is a *unary* operator, because it operates on a single value, in contrast to the *binary* comparison operators, which operate on two values. The operator is put in front of the operand, without any whitespace between them:
+
+```swift
+let asManyApplesAsOranges = false
+let notAsManyApplesAsOranges = !asManyApplesAsOranges // true
+```
```

Comment:
I think we need to more explicit with what's going on here using the unary operator. Stating something like follows:
Here we are declaring a variable `notAsManyApplesAsOranges` which ends up being a `true` value. Using the `!` operator before the `asManyApplesAsOranges` variable will return back to us the inverse of its current value. `asManyApplesAsOranges` remains to be `false` - we are NOT changing the value of the `asManyApplesAsOranges`, that original value remains.